### DISCUSSION TITLE
idea

### DISCUSSION THREAD

**Discussion by Kiger1989** [intellij-rust/intellij-rust#8646] [IntelliJ Rust Plugin Development]
System:win10
VS Version:2019
Idea Version: 
![image](https://user-images.githubusercontent.com/97730429/157205001-3738e6e9-ea94-40ac-a7a4-902280c6c3a4.png)
Plugins:
![image](https://user-images.githubusercontent.com/97730429/157205144-4ab4dde3-3132-42de-8938-432771bc87b1.png)
Error:
![image](https://user-images.githubusercontent.com/97730429/157205284-30290c89-1e72-4158-8e09-4ad40ad14655.png)
 Now,What should I do?

**Comment by Undin** [comment]
Could you specify what exactly doesn't work, please?
As far as I know, this error doesn't affect any Rust related things

**Reply by Kiger1989** [reply]
i want debug rust ,but not success, like this:
![image](https://user-images.githubusercontent.com/97730429/157572567-fcd3ca9a-c55f-4aaa-a745-436e9943baf3.png)

**Reply by Undin** [reply]
Which Rust version do you use? 1.59 or above?
If so, see https://github.com/intellij-rust/intellij-rust/issues/8526. In short, debugging should work as expected with Rust 1.58 or with the latest EAP versions of IDEA or CLion

**Comment by Undin** [comment]
JFTR `nightly-dev` channel is not official supported thing. It is used mostly for test purposes, so I'd recommend not using it. If you want to get the latest features/bug fixes, prefer [nightly](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-quick-start.html#install-nightly) channel instead
